git/src/lib.rs

```diff
@@ -21,8 +20,8 @@ pub enum Error {
         path: std::path::PathBuf,
     },
 
-    #[error("target `{0}` not found")]
-    TargetNotFound(Target),
+    #[error("target `{1}` not found in `{0}`")]
+    TargetNotFound(Box<gix::Url>, Target),
```

Comment:
Nitpick: should we use named fields/anonymous record, as is the case with e.g. `Self::Io`?